lib/location/location_utils.h

```diff
@@ -53,4 +53,12 @@ int location_utils_modem_params_read(struct location_utils_modem_params_info *mo
  */
 const char *location_utils_nrf_cloud_jwt_generate(void);
 
+/**
+ * @brief Get current system time and fill that into a given location_datetime.
+ *
+ */
+#include <modem/location.h>
```

Comment:
If this include is needed, I would move it to the beginning of the file.

Reply:
Done.